metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java

```diff
@@ -67,20 +70,27 @@ public MetafixLookupTest() {
     private static void setStubForWireMock() {
         WIRE_MOCK_SERVER.start();
 
-        final UrlPattern urlPattern = WireMock.urlPathEqualTo(RDF_PATH);
+        // stubs for RDF
+        UrlPattern urlPattern = WireMock.urlPathEqualTo(RDF_PATH);
         final String redirectToUrl = "/redirect" + RDF_PATH;
         final UrlPattern urlPatternRedirectToUrl = WireMock.urlPathEqualTo(redirectToUrl);
-
         WIRE_MOCK_SERVER.stubFor(WireMock.get(urlPattern)
             .willReturn(WireMock.temporaryRedirect(redirectToUrl)));
-
-        final String responseBody = new BufferedReader(new InputStreamReader(
-                    MetafixLookupTest.class.getResourceAsStream("." + RDF_PATH))).lines().collect(Collectors.joining("\n"));
-
+        String responseBody = loadFile(RDF_PATH);
         WIRE_MOCK_SERVER.stubFor(WireMock.get(urlPatternRedirectToUrl)
             .willReturn(WireMock.aResponse()
                 .withHeader("Content-Type", "text/turtle")
                 .withBody(responseBody)));
+
+        //stub for CSV
+        urlPattern = WireMock.urlPathEqualTo(CSV_PATH);
+        responseBody = loadFile(CSV_PATH);
```

Comment:
Why not give these variables appropriate names?

Reply:
OK, I've tried to do this by not reusing Objects. I hope that was your intention here.

Reply:
Yes, thanks, that's what I had in mind. Although I don't understand this part:

> also not reusing Objects.

Reply:
In https://github.com/metafacture/metafacture-fix/pull/316/commits/eef131aaba7cdfaae44a34e6deccaafc80386b56 both objects `urlPattern` and `responseBody` where reused. That made the naming rather generic.